src/webui/www/private/download.html

```diff
@@ -117,15 +117,17 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
                             <label for="dlLimit">QBT_TR(Limit download rate)QBT_TR[CONTEXT=HttpServer]</label>
                         </td>
                         <td>
-                            <input type="text" id="dlLimit" name="dlLimit" style="width: 16em;" placeholder="Bytes/s" />
+                            <input type="text" id="dlLimitText" style="width: 16em;" placeholder="KiB/s" />
+                            <input type="hidden" id="dlLimitHidden" name="dlLimit" />
```

Comment:
personally I would like hidden field comes first, so please move this line up and at other places too.